assets/js/googlesitekit/api/cache.js

```diff
@@ -22,12 +22,134 @@
 import { isEqual } from 'lodash';
 
 /**
- * Internal dependencies
+ * Prefix used for all Site Kit keys.
+ *
+ * Anything not using this key should not be touched by this library.
+ *
+ * @private
+ */
+export const STORAGE_KEY_PREFIX = 'googlesitekit_';
+
+const defaultOrder = [ 'localStorage', 'sessionStorage' ];
+let storageBackend;
+let storageOrder = [ ...defaultOrder	];
+/**
+ * Override the storage backend.
```

Comment:
The short descriptions should always start with a third-person verb.

Reply:
This has not been addressed here and in several other occurrences across this PR. E.g. it should be "Overrides..." here.